financial-connections/src/main/java/com/stripe/android/financialconnections/navigation/topappbar/TopAppBarHost.kt

```diff
@@ -0,0 +1,3 @@
+package com.stripe.android.financialconnections.navigation.topappbar
+
+internal interface TopAppBarHost
```

Comment:
This will receive a `updateTopAppBarElevation()` method in the next pull request to be informed about content scroll changes.